static/js/server_interaction.js

```diff
@@ -6,16 +6,14 @@
 
     //Connection information
     edx_adapt.apistr    = '/api/v1';
-    edx_adapt.server     = '52.210.53.36'
+    edx_adapt.server    = '52.210.53.36';
     edx_adapt.apiport   = '8080';
 
-    course_id = $(".xmodule_display", window.parent.document).attr('data-course-id');
+    edx_adapt.course_id = $(".xmodule_display", window.parent.document).data('course-id');
 
-    //Couurse string we will combine from course_id which is taken from the Course page
-    edx_adapt.coursestr = course_id.split(':')[1].split('+').join('/')
-
-    //course_id is automatically taken from the course page now it is shorted to course_name
-    edx_adapt.courseid  = /\+(.*?)\+/.exec(course_id)[1];
+    //Course string we will combine from course_id which is taken from the Course page for example
```

Comment:
`Convert course_id taken from the Course page into a course string compatible with EdxAdapt. E.g.